WSL_USBIP.md

```diff
@@ -110,43 +110,53 @@ uname -r
 
 The kernel can be found at: <https://github.com/microsoft/WSL2-Linux-Kernel>
 
-After finding the branch/tag that matches the version, clone that branch/tag.
+Clone the kernel repo, then checkout the branch/tag that matches your kernel version.
 
 ```bash
-sudo git clone https://github.com/microsoft/WSL2-Linux-Kernel.git /usr/src/5.4.72-microsoft-standard-WSL2 
-cd /usr/src/5.4.72-microsoft-standard-WSL2  
-git checkout linux-msft-5.4.72
+git clone https://github.com/microsoft/WSL2-Linux-Kernel.git 
+cd WSL2-Linux-Kernel  
+git checkout linux-msft-wsl-5.10.43.3
 ```
 
 Copy current configuration file.
 
 ```bash
-/usr/src/4.19.43-microsoft-standard$ sudo cp /proc/config.gz config.gz
-/usr/src/4.19.43-microsoft-standard$ sudo gunzip config.gz
-/usr/src/4.19.43-microsoft-standard$ sudo mv config .config
+cp /proc/config.gz config.gz
+gunzip config.gz
+mv config .config
 ```
 
-Run menuconfig to select kernel features to add. The number 4 is the number of cores I have.
+You may need to set CONFIG_USB=y in .config prior to running menuconfig to get all options enabled for selection.
+
+Run menuconfig to select kernel features to add.
 
 ```bash
 sudo make menuconfig
 ```
 
-Select desired features. In my case I selected USB/IP, VHCI HCD, Debug messages for USB/IP, USB Serial Converter Support.
+These are the necessary additional features in munconfig.  
```

Comment:
These lines have some trailing whitespace that the Markdown linter is complaining about.

Reply:
Interesting, the extra whitespace is to cause a line break which I thought was valid markdown.